packages/lit-analyzer/src/lib/analyze/default-lit-analyzer-context.ts

```diff
@@ -285,6 +285,6 @@ export class DefaultLitAnalyzerContext implements LitAnalyzerContext {
 
 		// Build a graph of component dependencies
 		const res = parseDependencies(file, this);
-		this.dependencyStore.importedComponentDefinitionsInFile.set(file.fileName, res);
+		this.dependencyStore.absorbComponentDefinitionsForFile(file, res);
```

Comment:
What's the difference between these methods?

Reply:
There is no difference except it looked like the intent was that `importedComponentDefinitionsInFile` should be `private`, with `absorbComponentDefinitionsForFile` being the public method to do this logic.

You can see the `dependencyStore` class defined here: https://github.com/runem/lit-analyzer/pull/338/files#diff-faaba94b3ffda920023b5e0ff3b54f7137199117142a418b7bd79df7e8951c5eR6-R9